src/commands/runtime/action/create.js

```diff
@@ -182,18 +186,32 @@ class ActionCreate extends RuntimeBaseCommand {
       if (limits) options.limits = limits
       if (paramsAction) options.parameters = paramsAction
       if (annotationParams) options.annotations = annotationParams
-
       const ow = await this.wsk()
-      const method = this.isUpdate() ? 'update' : 'create'
-      const result = await ow.actions[method]({ name, action: options })
-      if (flags.json) {
-        this.logJSON('', result)
-      }
+      await this.syncAction(ow, name, options, flags, method, super.logJSON.bind(this))
```

Comment:
nit: you don't really need to do the super/bind, you can just pass this.logJSON in javascript

Reply:
I kinda need to bind `this`. since the body of the function uses `this`. If I don't to it `this` it's undefined. 

```
logJSON (msg, obj) {
    if (msg) {
      this.log(msg, JSON.stringify(obj, null, 2))
    } else {
      this.log(JSON.stringify(obj, null, 2))
    }
  }
```

Reply:
this.logJSON(... works from anywhere in `ActionCreate` class .... its just the way inheritance works ...
if in doubt, please try it. 
otherwise, we would need things like:
```
const { args, flags } = super.parse.bind(this, ActionCreate)
// const { args, flags } = this.parse(ActionCreate)
```

Reply:
![image](https://user-images.githubusercontent.com/74616552/151121428-492d469a-803a-4545-808f-dae18f701eb6.png)
It's not me, it's just javascript

Reply:
Now I see your last commit. I was avoiding doing that so I can keep that function pure. But yeah, it works that way.